### ISSUE TITLE
Can this be compiled as a 64bit library, or not?

### ISSUE THREAD

**Issue by Lovewyrm** [xiaoyeli/superlu#32]
Can it even use 64 bit libraries?
Neither this or the MT version of this seems to look inside /usr/lib64, it's always /usr/lib/

Damnit, man. I just want to get another piece of software to run, which is also a pile of garbage.

This ancient archaic nonsense crap is always making me feel like a complete idiot.
I can use the damn library but compiling is, obviously, above me.

Pissed off.

**Comment by xiaoyeli** [comment]
Of course it can work with 64bit libraries.  If you don't have time or patience to read through the installation instruction, you certainly cannot use this library, then, stay away from it.

**Comment by Lovewyrm** [comment]
I have read the installation instructions, since the library was always compiled. 
I have also grepped all files for architecture hints, as well as library locations, as in, lib vs lib64. It merely did its best to appear like it compiled as a 32 bit library.

In reality, it did build as a 64bit library, but this was not obvious to me until I actually examined the file with objdump after  trying several times, believing it was 32 bit since it insisted on the /usr/lib/ path, something I have yet other projects I have built to do, they seem to find the right path.

Confusing.
